src/ReactNavigationCore.re

```diff
@@ -0,0 +1,5 @@
+[@bs.module "@react-navigation/core"]
+external stackRouter: ('a, 'b) => ReasonReact.reactElement = "StackRouter";
+
+[@bs.module "@react-navigation/core"]
+external createNavigator: ('a, 'b, 'c) => ReasonReact.reactElement = "";
```

Comment:
Do we need separate files for each of the bindings? I would say we can easily move them to a single "ReactNavigation.re" file

Reply:
For now, I want to separate it for a better view ;) 

I forget to add the WIP label because it's still in progress :)

Reply:
Cool, let me know when this PR is ready to be reviewed. Make sure to group it inside a single file (under multiple `modules` for better visibility). I think that's a better idea given the amount of code we have so far.

Reply:
I would actually even keep bindings along the code, just like on master, and move them out to a separate file when we actually need to reuse them (such as when we create TabNavigator).

I understand you might want to do it upfront tho.

Reply:
@grabbou I going to merge this PR because we need that example app. We have bindings inside one file so it's better than multifile. If we want in the next PR we can improve it.

Reply:
Yeah, single file is ok.